src/VisualStudio/Core/Def/Implementation/Experimentation/EnhancedColorExperiment.cs

```diff
@@ -96,6 +97,12 @@ private void UpdateThemeColors()
                 return;
             }
 
+            // We do not want to make any changes while in high contrast mode.
+            if (SystemParameters.HighContrast)
```

Comment:
Consider checking against `KnownColorThemes.HighContrast` instead, since the event you're subscribed to is about the VS theme changing.

Reply:
Pulling the theme id out of settings returns the last selected theme. Going to rely on windows for the time being.